zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -476,6 +480,42 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, region, infrastructu
     return entities
 
 
+def get_postgresql_cluster_members(kube_client, cluster_id, alias, region, infrastructure_account,
+                                   namespace=None):
+    entities = []
+
+    pods = get_all(kube_client, kube_client.get_pods, namespace)
+
+    for pod in pods:
+        obj = pod.obj
+
+        # TODO: filter in the API call
+        labels = obj['metadata'].get('labels', {})
+        if labels.get('application') != 'spilo' or labels.get('version') is None:
+            continue
+
+        pod_number = pod.name.split('-')[-1]
+        pod_namespace = obj['metadata']['namespace']
+        service_dns_name = '{}.{}.svc.cluster.local'.format(labels['version'], pod_namespace)
```

Comment:
I don't think an extra check here adds any value: an empty string is as bad as any other incorrect value, and that we can't check.